pull-request-suggester/processor/processor.go

```diff
@@ -75,7 +75,15 @@ func StartConcurrentProcessorClient(opt *ClientOptions) error {
 			continue
 		}
 		// Hooray we have a result back from the model
+		var cache []string
 		for _, f := range response.FileNameCommitIDPositions {
+			uniqkey := fmt.Sprintf("%s%s%d", f.CommitID, f.FileName, f.Position)
+			if !in(cache, uniqkey) {
+				cache = append(cache, uniqkey)
+			} else {
+				continue
+			}
```

Comment:
Hi...🦄 So I read your PR... Something looks weird here. Are you sure about this? 🌈